services/app-graphql/src/schema.graphql

```diff
@@ -758,63 +772,157 @@ enum FederalAuthority {
+    """
     managedCareEntities: [ManagedCareEntity!]!
+    """
+    The SPA or waiver(s) the state uses to deliver managed care to
+    the medicaid population. Options are: STATE_PLAN, WAIVER_1915B,
+    WAIVER_1115, VOLUNTARY, BENCHMARK, and TITLE_XXI
+    """
     federalAuthorities: [FederalAuthority!]!
+    """
+    If contract is in Lieu-of Services and Settings (ILOSs) 
+    in accordance with 42 CFR § 438.3(e)(2)
+    """
     inLieuServicesAndSettings: Boolean,
+    "If contract includes modifications to benefits provided by the managed care plans"
     modifiedBenefitsProvided: Boolean,
+    "If contract includes modifications to the geographic areas served by the managed care plans"
     modifiedGeoAreaServed: Boolean,
+    """
+    If contract includes modifications to the Medicaid beneficiaries served by the managed care
+    plans (e.g. eligibility or enrollment criteria)
+    """
     modifiedMedicaidBeneficiaries: Boolean,
+    "If contract includes modifications to the risk sharing strategy"
     modifiedRiskSharingStrategy: Boolean,
+    "If contract includes modifications to incentive arrangements"
     modifiedIncentiveArrangements: Boolean,
+    "If contract includes modifications to the withold agreements"
     modifiedWitholdAgreements: Boolean,
+    "If contract includes modifications to the state directed payments"
     modifiedStateDirectedPayments: Boolean,
+    "If contract includes modifications to the pass-through payments"
     modifiedPassThroughPayments: Boolean,
+    """
+    If contract includes modifications to payments to MCOs and PIHPs for enrollees that 
+    are a patient in an institution for mental disease
+    """
     modifiedPaymentsForMentalDiseaseInstitutions: Boolean,
+    "If contract includes modifications to the medical loss ratio standards"
     modifiedMedicalLossRatioStandards: Boolean,
+    """
+    If contract includes modifications to
+    other financial, payment, incentive or related contractual provisions
+    """
     modifiedOtherFinancialPaymentIncentive: Boolean,
+    "If contract includes modifications to the enrollment/disenrollment process"
     modifiedEnrollmentProcess: Boolean,
+    "If contract includes modifications to the grevience and appeal system"
     modifiedGrevienceAndAppeal: Boolean,
+    "If contract includes modifications to the network adequacy standards"
     modifiedNetworkAdequacyStandards: Boolean,
+    "If contract includes modifications to the length of the contract period"
     modifiedLengthOfContract: Boolean,
+    "If contract includes modifications to the non-risk payment arrangements"
     modifiedNonRiskPaymentArrangements: Boolean,
 }
 
+"Either the rate being created (NEW) or updating an existing rate (AMENDMENT)"
 enum RateType {
   NEW
   AMENDMENT
 }
 
+"""
+Determines how the capitation rate is established.
+With RATE_RANGE the state can choose to pay any amount 
+from the low to high end of the range
```

Comment:
```suggestion
With RATE_RANGE the state certifies that a range of rates
from the low to high end of the range as actuarially sound
```
would avoid saying the state "can choose" to do this, since this submission does not necessarily mean the state has been given approval to do so.